docs/docs/ref-01-top-level-api.md

```diff
@@ -66,9 +66,7 @@ factoryFunction createFactory(
 )
 ```
 
-Return a function that produces ReactElements of a given type. Like `React.createElement`,
-the type argument can be either an html tag name string (eg. 'div', 'span', etc), or a
-`ReactClass`.
+Return a function that produces ReactElements of a given type. Like `React.createElement`,the type argument can be either an html tag name string (eg. 'div', 'span', etc), or a `ReactClass`.
```

Comment:
i will do a revert, and separate en-docs and zh-docs commit.
for  en-docs ,the mainly modification were delete the unnecessary newline